10/buster-slim/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN buildDeps='xz-utils' \
       *) echo "unsupported architecture"; exit 1 ;; \
     esac \
     && set -ex \
-    && apt-get update && apt-get install -y ca-certificates curl wget gnupg dirmngr $buildDeps --no-install-recommends \
```

Comment:
I believe `ca-certificates` should not be in buildDeps.

Reply:
why not?

Reply:
if we _don't_ uninstall `ca-certificates` we're back at square one since it's `ca-certificates` which pull in `openssl`